### DISCUSSION TITLE
Calendar with timeOnly and mask

### DISCUSSION THREAD

**Discussion by TimmTha7er** [primefaces/community#616] [PrimeReact] [answered]
If I set timeOnly attribute to calendar with musk, keyboard input does not work correctly. I try to fix it using onChange and onBlur events(commented) but it seems like a crutch. How can i fix it properly?

Example: [CodeSandbox]( https://codesandbox.io/p/sandbox/primereact-demo-forked-kxwd42?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clps3dei600063b6indkk2wvr%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clps3dei500023b6iyacbpo0y%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clps3dei500033b6ix7vzd2y1%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clps3dei600053b6i54n0rbb5%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B60%252C40%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clps3dei500023b6iyacbpo0y%2522%253A%257B%2522id%2522%253A%2522clps3dei500023b6iyacbpo0y%2522%252C%2522tabs%2522%253A%255B%255D%257D%252C%2522clps3dei600053b6i54n0rbb5%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clps3dei500043b6ino376b80%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%2522%257D%255D%252C%2522id%2522%253A%2522clps3dei600053b6i54n0rbb5%2522%252C%2522activeTabId%2522%253A%2522clps3dei500043b6ino376b80%2522%257D%252C%2522clps3dei500033b6ix7vzd2y1%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clps3dei500033b6ix7vzd2y1%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

**Comment by melloware** [comment, marked as answer]
@TimmTha7er looks like already and open issue I would go comment and upvote it: https://github.com/primefaces/primereact/issues/4928

See also: https://github.com/primefaces/primereact/issues/4679
